front/src/modules/ui/board/components/BoardOptionsDropdownContent.tsx

```diff
@@ -1,7 +1,7 @@
-import { useRef, useState } from 'react';
+import { type Context, useRef, useState } from 'react';
```

Comment:
@thaisguigon I see that you are adding import type, which is quite new. What's the benefit of adding type import?  Is it related to:
https://stackoverflow.com/questions/61412000/do-i-need-to-use-the-import-type-feature-of-typescript-3-8-if-all-of-my-import

Is it something we want to use in the overall project? I don't see a direct benefit so as I would recommend here too to go for the simpler approach which is to not use these.